src/main/java/mate/academy/Main.java

```diff
@@ -1,9 +1,23 @@
 package mate.academy;
 
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    public static final Injector injector = Injector.getInstance("mate.academy");
+    public static final MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
+    public static final MovieSessionService movieSessionService =
+            (MovieSessionService) injector.getInstance(MovieSessionService.class);
+    public static final CinemaHallService cinemaHallService =
+            (CinemaHallService) injector.getInstance(CinemaHallService.class);
+
     public static void main(String[] args) {
         MovieService movieService = null;
```

Comment:
The `movieService` variable is being re-declared and initialized to `null`, which will lead to a `NullPointerException` when calling its methods. You should use the already initialized static final `movieService` instance declared at the class level.